src/frontend/src/pages/FinancePage/FinanceComponents/PendingAdvisorListModal.tsx

```diff
@@ -0,0 +1,110 @@
+import { Controller, useForm } from 'react-hook-form';
+import { useToast } from '../../../hooks/toasts.hooks';
+import { useSendPendingAdvisorList } from '../../../hooks/finance.hooks';
+import LoadingIndicator from '../../../components/LoadingIndicator';
+import NERFormModal from '../../../components/NERFormModal';
+import { Box } from '@mui/system';
+import * as yup from 'yup';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { TextField } from '@mui/material';
+import { ChangeEvent, useState } from 'react';
+
+const schema = yup.object().shape({
+  saboNumbers: yup.array().of(yup.number().required('SABO Number is required')).required('SABO Numbers are required')
+});
+
+interface PendingAdvisorModalProps {
+  open: boolean;
+  saboNumbers: number[];
+  onHide: () => void;
+}
+const PendingAdvisorModal: React.FC<PendingAdvisorModalProps> = ({ open, saboNumbers, onHide }) => {
+  const { mutateAsync: sendPendingAdvisorList, isLoading: sendPendingAdvisorListIsLoading } = useSendPendingAdvisorList();
+
+  const {
+    handleSubmit,
+    reset,
+    control,
+    formState: { errors }
+  } = useForm({
+    resolver: yupResolver(schema),
+    defaultValues: {
+      saboNumbers: saboNumbers
+    }
+  });
+
+  const toast = useToast();
+
+  if (sendPendingAdvisorListIsLoading) return <LoadingIndicator />;
```

Comment:
move this inside the modal and j disable the submit button while this is loading, it looks better